### ISSUE TITLE
model loss plateau's after 2 epochs

### ISSUE THREAD

**Issue by amygbAI** [ayulockin/SwAV-TF#14]
Hi, first off, thanks for the wonderful effort in converting this code from pyT to TF ..i was running your baseline model and looks like post epoch 2 where the loss is around 2.4 , the loss either doesn't go down or goes down almost negligibly till about epoch 10 .. and even at 10 its about 2.35/2.36 .. in your experiments have you noticed them go down in a 40 epoch run ? logically it doesn't make sense that once it plateaus more epochs can solve the problem ..

i am planning to use this for a totally different domain and dataset ( which of course, i will post on a public link AND ref ur work ) and would appreciate any thoughts

**Comment by sayakpaul** [comment]
Yeah we also experienced something similar. It can be because of numerous reasons:

* When you're changing the pretraining dataset usually other hyperparameters need to be changed as well. One of the most important component is the augmentation pipeline especially the crop ratio for random crops, strength of color distortion, and so on. We didn't run extensive experiments with these 
* You can refer to these guidelines from the original SwAV authors: https://github.com/facebookresearch/swav#common-issues. We tried some of these and got better results but didn't have the time to update things here. Sorry about that.

**Comment by amygbAI** [comment]
damn cool .. will do ..appreciate it
